content/english/blog/new_logo_2025.md

```diff
@@ -0,0 +1,24 @@
+---
+title: "PyPodcats Have a New Logo"
+meta_title: "PyPodcats Have a New Logo"
+description: "Finally after 1 year, we finally have a logo"
+date: 2025-06-17T05:00:00Z
+categories: ["Blog Post",]
+author: "Georgi"
+tags: ["python", "community", "women in tech", "women in Python", "women in open source","python underrespresnted groups"]
+draft: false
+image: "/images/logo-blog.png"
+---
+
+
+After a year of placeholder art and indecision, we finally have a real logo. And yes—it has a cat. On a mic. With a snake. We promise it makes sense.
+Our original stand-in was a charming sticker from Mariatta’s [“#TypoOfTheDay”](https://mariatta.ca/posts/typo_of_the_day/) series. It wasn’t meant to be a logo, but we were so eager to get *Hidden Figures of Python* out into the world that we just rolled with it. MVP: Minimum Viable Purr.<br>
```

Comment:
```suggestion
Our original stand-in was a charming sticker from Mariatta’s [“#TypoOfTheDay”](https://mariatta.ca/posts/typo_of_the_day/) series. It wasn’t meant to be a logo, but we were so eager to get *Hidden Figures of Python* out into the world that we just rolled with it. MVP: Minimum Viable Purr.


```

Add extra lines to make new paragraph. (instead of the `<br>` tag)